src/nevermined/Assets.ts

```diff
@@ -136,7 +136,8 @@ export class Assets extends Instantiable {
         nftAttributes: NFTAttributes | undefined,
         erc20TokenAddress: string | undefined,
         providers: string[] = [this.config.gatewayAddress],
-        txParams?: TxParameters
+        txParams?: TxParameters,
+        appId?: string
```

Comment:
maybe change order here so `txParams` is always the last parameter

Reply:
It would be better but the problem is that it introduces a breaking change